src/amber/cli/commands.cr

```diff
@@ -47,6 +48,7 @@ module Amber::CLI
       string ["-t", "--template"], desc: "# Preconfigure for selected template engine. Options: slang | ecr", default: "slang"
       string ["-d", "--database"], desc: "# Preconfigure for selected database. Options: pg | mysql | sqlite", default: "pg"
       string ["-m", "--model"], desc: "# Preconfigure for selected model. Options: granite | crecto", default: "granite"
+      string ["-r", "--recipe"], desc: "# Use a named recipe.  See documentation.", default: nil
```

Comment:
`see documentation` isn't a helpful message. Can this be something more like `see documentation at <amber web address>` or `run --recipe-help for more info` or something more prescriptive?